bgpd/bgp_vty.c

```diff
@@ -6433,11 +6438,20 @@ DEFUN (clear_bgp_instance_ipv6_safi_prefix,
        "IPv6 prefix\n")
 {
 	int idx_word = 3;
-	int idx_safi = 5;
-	int idx_ipv6_prefixlen = 7;
+	int idx_safi;
+	int idx_ipv6_prefix;
```

Comment:
The same here, both of these need to be set to zero.

Reply:
ok